app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -380,7 +388,7 @@ a.fold_element {
   }
 }
 
-div.content_editor.essence_date {
+.content_editor.essence_date {
```

Comment:
Selector `content_editor` should be written in lowercase with hyphens<br>Selector `essence_date` should be written in lowercase with hyphens